Purchases/Purchasing/TrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -17,6 +17,7 @@ import StoreKit
 class TrialOrIntroPriceEligibilityChecker {
 
     typealias ReceiveIntroEligibilityBlock = ([String: IntroEligibility]) -> Void
+    typealias ReceiveIntroEligibilityStatusBlock = (IntroEligibilityStatus) -> Void
```

Comment:
I think it'd be totally fine to skip this typealias, since the type itself is fairly simple